operatorconfig/moduleconfig/resiliency/v1.11.0/container-powermax-controller.yaml

```diff
@@ -14,8 +14,20 @@
 #
 #
 name: podmon
-image: dellemc/podmon:v1.11.0
+image: dellemc/podmon:v1.10.0
```

Comment:
Why the podman version is downgraded? Is it intentional?

Reply:
same as above comment